src/SignalR/clients/ts/signalr/src/WebSocketTransport.ts

```diff
@@ -80,7 +80,10 @@ export class WebSocketTransport implements ITransport {
                 // ErrorEvent is a browser only type we need to check if the type exists before using it
                 if (typeof ErrorEvent !== "undefined" && event instanceof ErrorEvent) {
                     error = event.error;
+                } else {
+                    error = "There was an error with the transport.";
```

Comment:
This would normally be an [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) instead of string when the event is an ErrorEvent, right? If so, we should update this.

```suggestion
                    error = new Error("There was an error with the transport.");
```